src/Admin/ArticleAdmin.php

```diff
@@ -91,6 +86,8 @@ public function toString($object)
     }
 
     /**
+     * @deprecated since 3.34, will be dropped in 4.0. Use TemplateRegistry services instead
```

Comment:
Version 3.34 of this bundle does not exist

Reply:
As this is still unreleased no need for a deprecation here